### PR TITLE
[RW-265] Content entity cloning

### PR DESCRIPTION
**Pull request by orakili**
Ticket: RW-265

This adds a module (that will be put on drupal.org) after review to enable cloning of content entities with the possibility to process the fields.

### CODE REVIEW

**Comment by orakili** [comment]
@attiks @lazysoundsystem Could you have a cursory look at the `content_entity_clone` module's code?

**Review by attiks — DISMISSED** [review]
Looking good

File: html/modules/custom/content_entity_clone/src/Form/BundleSettingsForm.php [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Entity\EntityTypeBundleInfoInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\content_entity_clone\Plugin\FieldProcessorPluginManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Cotent entity clone - bundle settings form handler.
```
Comment (lazysoundsystem) [inline html/modules/custom/content_entity_clone/src/Form/BundleSettingsForm.php:16]: Content

**Review by lazysoundsystem — DISMISSED** [review]
Haven't tested the functionality locally, but all looks very good

**Comment by orakili** [comment]
Thanks for the review and feedback. I moved the module to drupal.org. Merging.

**Pull request merged**